packages/anoncreds/tests/legacyAnonCredsSetup.ts

```diff
@@ -73,52 +65,9 @@ import {
 } from '../src'
 
 // Helper type to get the type of the agents (with the custom modules) for the credential tests
-export type AnonCredsTestsAgent =
-  | Agent<ReturnType<typeof getLegacyAnonCredsModules> & { mediationRecipient?: any; mediator?: any }>
-  | Agent<ReturnType<typeof getAskarAnonCredsIndyModules> & { mediationRecipient?: any; mediator?: any }>
-
-export const getLegacyAnonCredsModules = ({
-  autoAcceptCredentials,
-  autoAcceptProofs,
-}: { autoAcceptCredentials?: AutoAcceptCredential; autoAcceptProofs?: AutoAcceptProof } = {}) => {
-  const indyCredentialFormat = new LegacyIndyCredentialFormatService()
-  const indyProofFormat = new LegacyIndyProofFormatService()
-
-  // Register the credential and proof protocols
-  const modules = {
-    credentials: new CredentialsModule({
-      autoAcceptCredentials,
-      credentialProtocols: [
-        new V1CredentialProtocol({ indyCredentialFormat }),
-        new V2CredentialProtocol({
-          credentialFormats: [indyCredentialFormat],
-        }),
-      ],
-    }),
-    proofs: new ProofsModule({
-      autoAcceptProofs,
-      proofProtocols: [
-        new V1ProofProtocol({ indyProofFormat }),
-        new V2ProofProtocol({
-          proofFormats: [indyProofFormat],
-        }),
-      ],
-    }),
-    anoncreds: new AnonCredsModule({
-      registries: [new IndySdkAnonCredsRegistry()],
-    }),
-    dids: new DidsModule({
-      resolvers: [new IndySdkSovDidResolver(), new IndySdkIndyDidResolver()],
-      registrars: [new IndySdkIndyDidRegistrar()],
-    }),
-    indySdk: new IndySdkModule(getIndySdkModuleConfig()),
-    cache: new CacheModule({
-      cache: new InMemoryLruCache({ limit: 100 }),
-    }),
-  } as const
-
-  return modules
-}
+export type AnonCredsTestsAgent = Agent<
```

Comment:
Maybe we should loosen the any type restrictions within the `tests` folders.

Reply:
Yeah that might be a good one 👍